src/main/java/com/ticTacToc/Board.java

```diff
@@ -92,11 +92,13 @@ public void MakeMove(Player player) {
     }
 
     public void BoatMakeMove(Player player) {
```

Comment:
typo in Bot name

Reply:
Also this makemove method should go into the BotPlayer class since it belongs to him